contracts/payloads/IGP96/PayloadIGP96.sol

```diff
@@ -111,22 +133,84 @@ contract PayloadIGP95 is PayloadIGPMain {
         {
             // USDC-ETH DEX
             {
-                // USDC-ETH Dex
-                DexConfig memory DEX_USDC_ETH = DexConfig({
-                    dex: USDC_ETH_DEX,
-                    tokenA: USDC_ADDRESS,
-                    tokenB: ETH_ADDRESS,
-                    smartCollateral: true,
-                    smartDebt: false,
-                    baseWithdrawalLimitInUSD: 10_000, // $10k
-                    baseBorrowLimitInUSD: 0, // $0
-                    maxBorrowLimitInUSD: 1_020 // 0.4 ETH
-                });
-                setDexLimits(DEX_USDC_ETH); // Smart Collateral and Smart Debt
+                {
+                    // Set debt ceiling for USDC token
+                    AdminModuleStructs.UserBorrowConfig[]
+                        memory configsUSDC = new AdminModuleStructs.UserBorrowConfig[](
+                            1
+                        );
+
+                    configsUSDC[0] = AdminModuleStructs.UserBorrowConfig({
+                        user: USDC_ETH_DEX,
+                        token: USDC_ADDRESS,
+                        mode: 1,
+                        expandPercent: 1,
+                        expandDuration: 16777215,
+                        baseDebtCeiling: 1 * 1e6, // $1 in USDC
+                        maxDebtCeiling: 1000 * 1e6 // $1000 in USDC
+                    });
+
+                    LIQUIDITY.updateUserBorrowConfigs(configsUSDC);
+                }
+
+                {
+                    // Set debt ceiling for ETH token
+                    AdminModuleStructs.UserBorrowConfig[]
+                        memory configsETH = new AdminModuleStructs.UserBorrowConfig[](
+                            1
+                        );
+
+                    configsETH[0] = AdminModuleStructs.UserBorrowConfig({
+                        user: USDC_ETH_DEX,
+                        token: ETH_ADDRESS,
+                        mode: 1,
+                        expandPercent: 1,
+                        expandDuration: 16777215,
+                        baseDebtCeiling: (1 * 1e18) / (ETH_USD_PRICE / 1e2), // $1 in ETH
+                        maxDebtCeiling: (1000 * 1e18) / (ETH_USD_PRICE / 1e2) // $1000 in ETH
```

Comment:
simpler here with less risk for error would be, just set it to 0.4ETH and base debt ceiling to 0.001 ether